subprojects/base-services/src/main/java/org/gradle/internal/jvm/Jvm.java

```diff
@@ -359,7 +359,8 @@ private File findToolsJar(File javaHome) {
     }
 
     public boolean isIbmJvm() {
-        return false;
+        String vendor = System.getProperty("java.vm.vendor");
+        return vendor.toLowerCase().startsWith("ibm corporation");
```

Comment:
Note that this is probably outdated. J9 JVMs now use `Eclipse OpenJ9` as vendor. I am wondering if this logic is still used or relevant. I noticed there are decisions taken based on an IBM JVM, but this most likely is not correctly identified anymore.